app/api/auth/check-banned/route.ts

```
@@ -0,0 +1,70 @@
+        },
+      },
+    });
+
+    return NextResponse.json({
+      isBanned: (user?.teamBans && user.teamBans.length > 0) || false,
+      bannedTeams:
+        (user?.teamBans &&
+          user.teamBans.map((ban: TeamBan & { team: { slug: string } }) => ({
+            id: ban.teamId,
+            slug: ban.team.slug,
+          }))) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)